src/dictionaries/words

```diff
@@ -0,0 +1,205 @@
+app
+apps
+args
+auth
+authenticator
+bar
+baz
+behavior
+behaviour
```

Comment:
Does the dictionary have locale variants?  Otherwise I imagine it would either allow British and American spellings, or enforce American?

Reply:
At the moment Tim has hardcoded it to `en_NZ` (I don't know what the differences to `en_GB` are)

Reply:
I've deliberately allowed both for now. 

I think we should come to a team decision and pick one as follow up - see comments on "Parametrize". 

Since it's not a keyword AFAIK in any languages we use (my reason for "color", used in HTML to use US spelling) I think "u" should be in there.